docs/schema_registry_tutorial.rst

```diff
@@ -0,0 +1,532 @@
+Then the consumer can read records where the Kafka value is of `Payment` class.
+To ensure that the object is deserialized using the application's code-generated `Payment` class, configure the deserializer to use Avro `SpecificRecord`, i.e., ``SPECIFIC_AVRO_READER_CONFIG`` should be set to _true_.
+For example:
+
+.. sourcecode:: java
+
+   import io.confluent.kafka.serializers.KafkaAvroDeserializer;
+   import io.confluent.kafka.serializers.AbstractKafkaAvroSerDeConfig;
+
+   ...
+   props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, KafkaAvroDeserializer.class);
+   props.put(KafkaAvroDeserializerConfig.SPECIFIC_AVRO_READER_CONFIG, true); 
+   props.put(AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG, schemaRegistryUrl);
+   ...
+
+   ...
+   KafkaConsumer<String, Payment> consumer = new KafkaConsumer<>(props));
+   consumer.subscribe(Collections.singletonList(TOPIC));
+   while (true) {
+     ConsumerRecords<String, Payment> records = consumer.poll(100);
+     for (ConsumerRecord<String, Payment> record : records) {
+       String key = record.key();
+       Payment value = record.value();
+     }
+   }
+   ...
+
+For a full Java consumer example, refer to `the consumer example <https://github.com/confluentinc/examples/blob/5.0.0-post/clients/avro/src/main/java/io/confluent/examples/clients/basicavro/ConsumerExample.java>`_.
+Because the `pom.xml` includes ``avro-maven-plugin``, the `Payment` class is automatically generated during compile.
+To run this consumer, first compile the project and then run ``ConsumerExample`` (assuming you already ran the ``ProducerExample`` above).
+
+.. sourcecode:: bash
+
+   $ mvn clean compile package
+   $ mvn exec:java -Dexec.mainClass=io.confluent.examples.clients.basicavro.ConsumerExample
+
+You should see:
+
+.. sourcecode:: bash
+
+   ...
+   offset = 0, key = id0, value = {"id": "id0", "amount": 1000.0}
+   offset = 1, key = id1, value = {"id": "id1", "amount": 1000.0}
+   offset = 2, key = id2, value = {"id": "id2", "amount": 1000.0}
+   offset = 3, key = id3, value = {"id": "id3", "amount": 1000.0}
+   offset = 4, key = id4, value = {"id": "id4", "amount": 1000.0}
+   offset = 5, key = id5, value = {"id": "id5", "amount": 1000.0}
+   offset = 6, key = id6, value = {"id": "id6", "amount": 1000.0}
+   offset = 7, key = id7, value = {"id": "id7", "amount": 1000.0}
+   offset = 8, key = id8, value = {"id": "id8", "amount": 1000.0}
+   offset = 9, key = id9, value = {"id": "id9", "amount": 1000.0}
+   ...
+
+Hit ``Ctrl-C`` to stop.
+
+
+Other Kafka Clients
+^^^^^^^^^^^^^^^^^^^
+
+The objective of this tutorial is to learn about Avro and |sr| centralized schema management and compatibility checks.
+To keep examples simple, this tutorial focuses on Java producers and consumers, but other Kafka clients work in similar ways.
+For examples of other Kafka clients interoperating with Avro and |sr|:
```

Comment:
nit: `For examples of other Kafka clients interoperating with Avro and Schema Registry [see]:` ?